drivers/kw2xrf/kw2xrf_intern.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2016 PHYTEC Messtechnik GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     drivers_kw2xrf
+ * @{
+ * @file
+ * @brief       Internal function of kw2xrf driver
+ *
+ * @author      Johann Fischer <j.fischer@phytec.de>
+ * @}
+ */
+
+#include "panic.h"
+#include "kw2xrf.h"
+#include "kw2xrf_spi.h"
+#include "kw2xrf_reg.h"
+#include "kw2xrf_getset.h"
+#include "kw2xrf_intern.h"
+#include "overwrites.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+void kw2xrf_disable_interrupts(kw2xrf_t *dev)
+{
+    /* Clear and disable all interrupts */
+    kw2xrf_write_dreg(dev, MKW2XDM_PHY_CTRL2, 0xff);
+    int reg = kw2xrf_read_dreg(dev, MKW2XDM_PHY_CTRL3);
+    reg |= MKW2XDM_PHY_CTRL3_WAKE_MSK | MKW2XDM_PHY_CTRL3_PB_ERR_MSK;
```

Comment:
ah the following line 

```
kw2xrf_write_dreg(dev, MKW2XDM_PHY_CTRL3, 0x03);
```

here actually `0x03` should be `reg` because `MKW2XDM_PHY_CTRL3_WAKE_MSK | MKW2XDM_PHY_CTRL3_PB_ERR_MSK = 0x03` according to the defines.

so fixing this by either remove `reg` or better as `0x03` is some magic number with
```
kw2xrf_write_dreg(dev, MKW2XDM_PHY_CTRL3, reg);
```

should do